### DISCUSSION TITLE
Network guide

### DISCUSSION THREAD

**Discussion by steveodhiambo** [tendermint/tendermint#8045] [General]
Hello, I had an issue on how to send requests to nodes that I created using the network guide in the docs that talks about **setting up config files for 5 validators and 3 non-validators** [(Docs)](https://docs.tendermint.com/v0.35/networks/docker-compose.html). 
Do the nodes already have the Kvstore app already implemented?

**Comment by creachadair** [comment]
> Do the nodes already have the Kvstore app already implemented?

A stock build of Tendermint core has the `kvstore` app built in, you can enable it by specifying `--proxy-app=kvstore` as an argument to the `start` command.
